src/plugins/mer/merabstractvmstartstep.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
 **
-** Copyright (C) 2014 Jolla Ltd.
+** Copyright (C) 2014-2015,2019 Jolla Ltd.
```

Comment:
Check https://www.gnu.org/licenses/gpl-howto.html

> For software with several releases over multiple years, it's okay to use a range (“2008-2010”) instead of listing individual years (“2008, 2009, 2010”) if and only if every year in the range, inclusive, really is a “copyrightable” year that would be listed individually; and you make an explicit statement in your documentation about this usage.

Reply:
That howto talks about releases. I don't think they mean that each and every file has to have modifications during the year. But sure, using individual years is definitely correct.